scss/_patterns_strip.scss

```diff
@@ -1,24 +1,26 @@
 // Full width strip pattern
 @mixin vf-p-strip {
 
-  %strip {
-    background-color: transparent;
-    clear: both;
-    width: 100%;
-  }
-
-  .p-strip {
-    @extend %strip;
-
-    &--light {
-      @extend %strip;
-      background-color: $color-light;
+    // strip
+    %strip {
+      background-color: transparent;
+      clear: both;
+      padding: 2rem 0;
+      width: 100%;
     }
 
-    &--dark {
+    .p-strip {
       @extend %strip;
-      background-color: $color-dark;
-      color: $color-x-light;
+
+      &--light {
+        @extend %strip;
+        background-color: #f7f7f7;
```

Comment:
Doh - copied from your codepen and forgot these were swapped out in my original 🙄